tests/common/fixtures/ptfhost_utils.py

```diff
@@ -228,8 +228,9 @@ def ptf_portmap_file_module(rand_selected_dut, ptfhost, tbinfo):
 def run_icmp_responder_session(duthosts, duthost, ptfhost, tbinfo):
     """Run icmp_responder on ptfhost session-wise on dualtor testbeds with active-active ports."""
     # No vlan is available on non-t0 testbed, so skip this fixture
-    if "mixed" not in tbinfo["topo"]["name"]:
-        logger.info("Skip running icmp_responder at session level, it is only for dualtor-mixed testbed.")
+    if "mixed" not in tbinfo["topo"]["name"] and "aa" not in tbinfo["topo"]["name"]:
```

Comment:
`"aa" not in tbinfo["topo"]["name"]:` -> '"dualtor-aa" not in tbinfo["topo"]["name"]:' ?